benchmarks_runner/eval.py

```diff
@@ -81,29 +82,31 @@ def evaluate_results(
                     message = response.get('message', None)
                     if message is None:
                         # most likely came from the ARS
-                        message = ((response.get('fields') or {}).get('data') or {}).get('message', {})
+                        message = reduce(lambda val, key: val.get(key) if val else {}, ["fields", "data", "message"], response)
```

Comment:
This line returns `None` for `{"fields": {"data": {"message": None}}}` and `{"fields": {"data": {}}}`.

Looks like we need `message` to be a dict. I think this version covers those situations.
```suggestion
                        message = reduce(lambda val, key: {} if val.get(key) is None else val[key], ["fields", "data", "message"], response)
```